kindlecomicconverter/comic2ebook.py

```diff
@@ -1222,6 +1224,58 @@ def checkPre(source):
         raise UserWarning("Target directory is not writable.")
 
 
+def makeFusion(sources, qtgui=None):
+    filepath = []
+    GUI = qtgui
+    start = perf_counter()
+    pageTracker : int = 0
+    combinePath : str = os.path.join(os.path.dirname(sources[0]), "combinationTemp")
+    os.mkdir(combinePath)
+    print("Running Fusion")
+        
+    for source in sources:
+        print(f"Processing {source}...")
+        
+        checkPre(source)
+        print("Checking images...")
+        path : str = getWorkFolder(source)
```

Comment:
I think it would be much simpler to instead of calling `getWorkFolder()`, to instead write a simple function that is similar to `getWorkFolder()` that just moves/extracts all the images into one folder.

E.g.

If filefusion is checked:
1) create a temp folder same name as first source (the fusion folder)
2) move/extract all the files to be fused into individual folders inside the fusion folder

Example:
I start with `TITLE - ch1.cbz` and `TITLE - ch2.cbz`
I checked the fusion box.
KCC should create a **folder** called `TITLE -ch1-fused` with 2 subfolders inside with each chapter extracted. 

And then passes that folder through the normal process.